whatever_test_sbcm/test it man - sbcm.ipynb

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
<p>Panie, co to ma być?!</p>

 _Reply via <a href='https://app.reviewnb.com/slomeks/oku_notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>